patches/components-content_settings-core-common-cookie_settings_base.cc.patch

```diff
@@ -1,20 +1,28 @@
 diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
-index f0026d1c140325735e431e4578e8da5c5513c4ea..950b1ddf3646a2a87cda2f5a15a08ffb205343aa 100644
+index 88d12bf5e10ad48b713d99ba9dcafea8d06b02de..4fff7b1b3663a9d78655e59c03e987e7a4f2e67f 100644
 --- a/components/content_settings/core/common/cookie_settings_base.cc
 +++ b/components/content_settings/core/common/cookie_settings_base.cc
-@@ -544,6 +544,7 @@ CookieSettingsBase::DecideAccess(
+@@ -538,6 +538,7 @@ CookieSettingsBase::DecideAccess(
      return BlockAllCookies{};
    }
  
 +  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
    if (!is_third_party_request) {
      return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
    }
-@@ -649,6 +650,7 @@ CookieSettingsBase::GetCookieSettingInternal(
+@@ -647,6 +648,7 @@ CookieSettingsBase::GetCookieSettingInternal(
    ContentSetting cookie_setting = GetContentSetting(
        url, first_party_url, ContentSettingsType::COOKIES, &setting_info);
  
 +  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
    bool is_explicit_setting = !setting_info.primary_pattern.MatchesAllHosts() ||
                               !setting_info.secondary_pattern.MatchesAllHosts();
- 
\ No newline at end of file
+ 
+@@ -712,6 +714,7 @@ CookieSettingsBase::GetCookieSettingInternal(
+     CHECK(is_third_party_request, base::NotFatalUntil::M128);
+     CHECK(global_setting_or_embedder_blocks_third_party_cookies,
+           base::NotFatalUntil::M128);
++    if (false)
```

Comment:
this patch doesn't follow patching conventions, but I'm concerned that we're running into this in the first place. Also it says it's not fatal yet so why do we need to skip it?

Reply:
I can correct the patch, but I may need @goodov to chime in on why this would be hit. I believe that even though it's marked as `NotFatalUntil::M128`, it was causing our cookie tests to fail.

Reply:
part of the problem with a patch like this is that there is no place to provide any context for this. The commit message just says fix the CHECK, but I think it's important to explain why we're hitting the CHECK if we expect this to happen

Reply:
Fixed in d30b9de179055404a4f682103689466c2a5a5de2